grin/src/Grin/Pretty.hs

```diff
@@ -84,28 +85,31 @@ instance Pretty Name where
 --  nice colors for syntax highlight
 --  better node type syntax (C | F | P)
 
+-- | Print a given expression with highlighted external functions.
+prettyWithExternals :: [External] -> Exp -> Doc
```

Comment:
The name is confusing. This function does what the comment states, but the name is misleading. `prettyHighlightExternals` would be better.

Reply:
As mentioned later the `prettyPrintProgram` would be even better.

Reply:
Done.